### DISCUSSION TITLE
copy graphic elements but not text

### DISCUSSION THREAD

**Discussion by lbr991** [pymupdf/PyMuPDF#2640] [Looking for help] [answered]
I'd like to to copy the graphic elements of a pdf page by page (let's say a banner or background image etc), but without the text.
Is there a way to do this?

**Comment by JorjMcKie** [comment, marked as answer]
This is possible - at least mostly.
If the desired object is an image, this is particularly easy:
1. identify the image's xref and its bbox
2. Use `page.insert_image(bbox, xref=xref, overlay=False)`

If it is a drawing (vector graphics), things are more difficult. You first must find a way to identify the object. It will generally be a sublist of `page.get_drawings()`. Once identified, just re-execute the draw commands contained in the sublist of paths.

If it is text, then things are similar as with vector graphics.
So much for now ...
